audio/speech/getting-started/get_started_with_chirp_2_sdk.ipynb

```diff
@@ -0,0 +1,1331 @@
+        ")"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "agk_M0xRwzv0"
+      },
+      "outputs": [],
+      "source": [
+        "for audio_sample_bytes, transcription in streaming_recognize_output:\n",
+        "    print_transcription(audio_sample_bytes, transcription)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oYCgDay2hAgB"
+      },
+      "source": [
+        "### Perform real-time speech recognition"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "F83r9aiNhAgD"
+      },
+      "source": [
+        "#### Define real-time recognition configuration\n",
+        "\n",
+        "As for the streaming transcription, you define the real-time recognition configuration which allows you to set the model to use, language code of the audio and more."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "j0WprZ-phAgD"
+      },
+      "outputs": [],
+      "source": [
+        "real_time_config = cloud_speech.RecognitionConfig(\n",
+        "    language_codes=[\"en-US\"],\n",
+        "    model=\"chirp_2\",\n",
+        "    features=cloud_speech.RecognitionFeatures(\n",
+        "        enable_automatic_punctuation=True,\n",
+        "    ),\n",
+        "    auto_decoding_config=cloud_speech.AutoDetectDecodingConfig(),\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "r2TqksAqhAgD"
+      },
+      "source": [
+        "#### Define the real-time request configuration\n",
+        "\n",
+        "Next, you define the real-time request passing the configuration and the audio sample you want to transcribe. Again, you don't need to define a recognizer."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "Nh55mSzXhAgD"
+      },
+      "outputs": [],
+      "source": [
+        "real_time_request = cloud_speech.RecognizeRequest(\n",
+        "    recognizer=f\"projects/{PROJECT_ID}/locations/{LOCATION}/recognizers/_\",\n",
+        "    config=real_time_config,\n",
+        "    content=short_audio_sample_bytes,\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "817YXVBli0aY"
+      },
+      "source": [
+        "#### Run the real-time recognition request\n",
+        "\n",
+        "Finally you submit the real-time recognition request."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "rc0cBrVsi7UG"
+      },
+      "outputs": [],
+      "source": [
+        "response = client.recognize(request=real_time_request)\n",
+        "\n",
+        "real_time_recognize_results = parse_real_time_recognize_response(response)"
```

Comment:
Since these parsing functions only seem to be used for visualizing the output in a notebook, I wonder if it's possible to change the `print_transcription()` method to call `parse_real_time_recognize_response()` or the appropriate method to make it simpler?

Reply:
I see. I didn't put everything in one function because I want to describe steps you might cover for getting the output from the API.